test/src/test/java/org/corfudb/integration/LogReplicationAbstractIT.java

```diff
@@ -716,33 +719,31 @@ public void verifyDataOnStandby(int expectedConsecutiveWrites) {
     }
 
     public void verifyInLogEntrySyncState() throws InterruptedException {
-        LogReplicationMetadata.ReplicationStatusKey key =
-                LogReplicationMetadata.ReplicationStatusKey
-                        .newBuilder()
-                        .setClusterId(DefaultClusterConfig.getStandbyClusterId())
-                        .build();
+        ReplicationStatusKey key = ReplicationStatusKey.newBuilder()
+                .setClusterId(DefaultClusterConfig.getStandbyClusterId())
+                .build();
 
-        LogReplicationMetadata.ReplicationStatusVal replicationStatusVal = null;
+        ReplicationStatusVal replicationStatusVal = null;
 
-        while (replicationStatusVal == null || !replicationStatusVal.getSyncType().equals(LogReplicationMetadata.ReplicationStatusVal.SyncType.LOG_ENTRY)
-                || !replicationStatusVal.getSnapshotSyncInfo().getStatus().equals(LogReplicationMetadata.SyncStatus.COMPLETED)) {
+        while (replicationStatusVal == null || !replicationStatusVal.getSyncType().equals(ReplicationStatusVal.SyncType.LOG_ENTRY)
```

Comment:
nit: Is this indentation ok?

Reply:
yes.  4 space indentation and 8 for the next line